composer.json

```diff
@@ -93,7 +93,14 @@
         "patches": {
```

Comment:
One more thing i noticed just now, when I build composer I get a new folder
<img width="341" alt="image" src="https://user-images.githubusercontent.com/11507647/176872008-10e2df7f-04c5-4fe1-b742-4038aad4ea35.png">
pls look into it

Reply:
![image](https://user-images.githubusercontent.com/6751237/176906062-fa899a60-9c09-47a7-b4c4-46b688210471.png)

Reply:
we will probably gitignore